packages-web/calendar/tests/e2e/specs/renderDifferentViews.spec.ts

```diff
@@ -0,0 +1,26 @@
+import page from "../../../../../configs/e2e/src/pages/page";
+import Calendar from "../objects/calendar.widget";
+
+describe("Calendar", () => {
+    const calendar = new Calendar("calendar1");
+
+    it("renders month view", () => {
+        page.open();
```

Comment:
It's better to place this in the describe. If this test gets deleted for some reasons, the other tests will fail as well.

```
page.open();
const calendar = new Calendar("calendar1");
```

Reply:
open page moved to beforeAll